src/woo_search/conf/base.py

```diff
@@ -217,34 +215,32 @@
 )
 REST_FRAMEWORK["EXCEPTION_HANDLER"] = "rest_framework.views.exception_handler"
 
+API_VERSION = "0.1.0"
+
 SPECTACULAR_SETTINGS = {
     "SCHEMA_PATH_PREFIX": "/api/v1",
     "TITLE": "WOO Search",
-    "DESCRIPTION": dedent(
+    "DESCRIPTION": _(
         """
-        The WOO Search API provides an interface to the underlying Elastic Search index.
-
-        Through this API, it's possible to add published documents to the API, or to
-        schedule removal of a publication in case it's being retracted. When a supported
-        file type is provided, the content of the document itself will also be indexed.
-
-        **Authentication**
+## Documentation
 
-        TODO - not implemented yet!
+The WOO Search API provides an interface to the underlying Elastic Search index.
 
-        This API makes use of API Key authentication. All requests must have the
-        `Authorization: token <token-value>` HTTP header, where `<token-value>` is
-        replaced with the actual API key. API keys are managed in the admin interface,
-        and you obtain one by contacting the administrator(s).
+Through this API, it's possible to add published publications/documents to the API, to
+schedule removal of a publication/documents in case it's being retracted/deleted and to search
+indexed publication/documents based on certain search criteria.
+When a supported file type (files supported by [Apache Tika](https://tika.apache.org/1.10/formats.html))
+is provided during the document index, the content of the document itself will also be indexed.
+So it can be used when searching for documents.
 
-        **Audit logging**
+**Authentication**
 
-        TODO - not implemented yet!
+This API makes use of API Key authentication. All requests must have the
+`Authorization: token <token-value>` HTTP header, where `<token-value>` is
+replaced with the actual API key. API keys are managed in the admin interface,
+and you obtain one by contacting the administrator.
 
```

Comment:
Do we want some mention about needing to configure the GPP-Publicatiebank? Or will we do that in a different location?

Reply:
I think that belongs in the GPP-publicatiebank docs. Ultimately for GPP-Zoeken,  we don't particularly care where the data is coming from, as long as we can authenticate to download the documents.